homeassistant/components/plugwise/number.py

```diff
@@ -36,15 +36,24 @@ class PlugwiseNumberEntityDescription(
 ):
     """Class describing Plugwise Number entities."""
 
+    native_max_value_key: str | None = None
+    native_min_value_key: str | None = None
+    native_step_key: str | None = None
+    native_value_key: str | None = None
```

Comment:
These aren't optional at this point? So initializing and typing them as `None` seems incorrect. Let's type them as `str` and make them required.

Reply:
No, they are not optional. Will change them as suggested.

Reply:
Implemented as suggested.